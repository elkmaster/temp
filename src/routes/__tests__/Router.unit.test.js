import React from 'react';
import Router from '../index';
import { shallow } from 'enzyme';

describe('Router component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Router />);
    expect(wrapper).toMatchSnapshot();
  });
});
