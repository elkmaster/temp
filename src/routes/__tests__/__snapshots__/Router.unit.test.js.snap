// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Router component should render correctly 1`] = `
<div>
  <Switch>
    <PublicRoute
      component={[Function]}
      exact={true}
      path="/"
    />
    <PublicRoute
      component={[Function]}
      path="/comments"
    />
    <PublicRoute
      component={[Function]}
      path="/login"
    />
    <Connect(PrivateRoute)
      component={[Function]}
      path="/cabinet"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</div>
`;
